portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,6 +29,7 @@
   String messages[] =
       new String[] {"This is amazing!", "Thank you for your time!", "Nice to meet you!"};
   private List<String> comments = Arrays.asList(messages);
+  
```

Comment:
Are there any whitespaces here? It might be why the lint check is complaining :). We do not put any spaces in empty lines.

Reply:
That was the problem indeed. There was a whitespace on that line. Thank you very much